roles/runtime/tasks/initialize_teardown.yml

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 - name: Refresh Environment Info with Descendants
+  when: no
```

Comment:
see other comment about changing env_info instead as a single point of change fix.